CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ project(F3D
   DESCRIPTION "F3D - A fast and minimalist 3D viewer"
   LANGUAGES C CXX)
 
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
Can we just set `CXX_STANDARD` on each target instead? Idk if doing that is good practice or not.

Reply:
I was not able to make it work for some reason, ill try again